addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -204,10 +204,16 @@ export class Many2XAutocomplete extends Component {
     }
 
     getCreationContext(value) {
-        return makeContext([
+        const ctx = makeContext([
             this.props.context,
             { [`default_${this.props.nameCreateField}`]: value },
         ]);
+        if (this.env.model.root && this.env.model.root.records) {
```

Comment:
Hi @FrancoisGe - I don't like this solution, and I expect the branch to go red.
Ideally this change should be in relational_model or list_controller/renderer, but I'm not sure
Basically when multiple records are selected, the `active_ids` in the context are not correct
(the action seems to handle it well https://github.com/odoo/odoo/blob/16.0/addons/web/static/src/webclient/actions/action_service.js#L299)